rasa/nlu/tokenizers/spacy_tokenizer.py

```diff
@@ -34,6 +34,8 @@ def get_doc(self, message: Message, attribute: Text) -> "Doc":
 
     def tokenize(self, message: Message, attribute: Text) -> List[Token]:
         doc = self.get_doc(message, attribute)
```

Comment:
the return type of `Doc` should be `Optional["Doc"]`, shouldn't it?

Reply:
Yep, changed it!